tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -248,20 +248,6 @@ def get_to_addrs_email_messages_map(self, sqs_message):
         # eg: { ('milton@initech.com', 'peter@initech.com'): mimetext_message }
         return to_addrs_to_mimetext_map
 
-    def send_smtp_email(self, smtp_server, message, to_addrs):
-        smtp_port = int(self.config.get('smtp_port', 25))
-        smtp_ssl = bool(self.config.get('smtp_ssl', True))
```

Comment:
instead let's move the config here to __init__ for email delivery so its one time setup, basically what you were doing in the provider specific subclasses.